project/profiles/capacity4more/modules/c4m/content/c4m_content_book/includes/c4m_content_book_plugin_argument_default.inc

```diff
@@ -19,21 +19,33 @@ class c4m_content_book_plugin_argument_default extends views_plugin_argument_def
   /**
    * Return the node ID of the help pages book.
    *
-   * @return int
+   * @return bool|int
    *   Return the node ID of the help pages book.
    */
   public function get_argument() {
     // Get the node ID of the Help & Guidance book.
     $help_book_title = variable_get('c4m_help_book_title', 'Help & Guidance');
     $query = new EntityFieldQuery();
-    $entities = $query->entityCondition('entity_type', 'node')
+    $result = $query->entityCondition('entity_type', 'node')
       ->propertyCondition('type', 'book')
       ->propertyCondition('title', $help_book_title)
       ->propertyCondition('status', NODE_PUBLISHED)
       ->propertyOrderBy('nid')
       ->range(0, 1)
       ->execute();
-    $entity = array_keys($entities['node']);
+
+    // If the Help & Guidance book ID could not be determined, add a human
+    // friendly error message and return early -which will cause Drupal to
+    // display `The requested page could not be found`.
+    if (!isset($result['node'])) {
+      drupal_set_message(t('Unable to retrieve help pages. Please <a href="@contact-page">contact the site administrator</a>.', array(
+        '@contact-page' => url('contact', array('purl' => array('disabled' => TRUE))),
```

Comment:
@ybaras 
take the whole link out of the `t()` function and use a variable.
you can use `l()` to create the anchor element